### PR TITLE
Fix typo in pmem calculation

### PR DESCRIPTION
**Pull request by asmacdo**
Fixes https://github.com/con/duct/issues/150

Also
 - use floats rather than ints for 0
 - add % to human readable summary

### CODE REVIEW

**Review by yarikoptic — CHANGES_REQUESTED** [review]
add a test . 
could mimic original invocation which observed that odd behavior: run `duct duct sleep 1` with some short periods for sampling/report

**Comment by yarikoptic** [comment]
tests aren't happy , hadn't looked in detail yet.

**Comment by asmacdo** [comment]
All the MacOS tests are failing because we cannot collect samples. The SANITY commit is not intended to be merged, its to show that there was plenty of time to collect a sample. This is related to https://github.com/con/duct/issues/82

**Comment by asmacdo** [comment]
From one of the failing mac runs *Samples Collected: 0*, most likely because `ps -s` isnt an option.
```
Wall Clock Time: 1.139 sec
Memory Peak Usage (RSS): unknown bytes
Memory Average Usage (RSS): unknown bytes
Virtual Memory Peak Usage (VSZ): unknown bytes
Virtual Memory Average Usage (VSZ): unknown bytes
Memory Peak Percentage: unknown%
Memory Average Percentage: unknown%
CPU Peak Usage: unknown%
Average CPU Usage: unknown%
Samples Collected: 0
Reports Written: 0
```

**Comment by asmacdo** [comment]
image

The tests didnt run, looks like a partial outage

File: .github/workflows/test.yaml [hunk 1]
```diff
@@ -26,7 +26,7 @@ jobs:
       fail-fast: false
       matrix:
         os:
-          - macos-12
+          # - macos-12
           - macos-latest
```
Comment (yarikoptic) [inline .github/workflows/test.yaml:30]:
```suggestion
          # - macos-latest
```

since unlikely to work there either

**Pull request merged**